doc/news/_preparation_next_release.md

```diff
@@ -226,6 +226,9 @@ Many problems were resolved with the following fixes:
 - The [`list` plugin](http://libelektra.org/plugins/list) which is responsible
   for global mounting had a bug which prevented globally mounted plugins from
   being configurable. *(Thomas Wahringer)*
+- Fix an issue with excluded plugins for rest-backend builds with ASAN enabled *(Lukas Winkler)*
+  - Reenable `ni` plugin for ASAN builds
+  - Disable failing markdown parser test for `validation` since it leaks memory and thus fails
```

Comment:
Typo: `markdown parser` → `Markdown Shell Recorder`

Reply:
I think it would make sense to also add the information, that we only disable the test for ASAN enabled builds.

Reply:
I agree!